src/plugins/cpptemplate/README.md

```diff
@@ -0,0 +1,17 @@
+- infos = Information about the cpptemplate plugin is in keys below
+- infos/author = Author Name <elektra@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = getstorage setstorage
+- infos/status = maintained unittest nodep nodoc concept
```

Comment:
These defaults are not good and inconsistent with the other template.

Reply:
That is not a problem in my opinion since `copy-template` uses `README.md` of the `template` plugin anyway.